### PR TITLE
Move 'Setting up Sailfish' docs into 'Managing Sailfish OS'. Set redi…

### PR DESCRIPTION
**Pull request by jovirkku**
…rection from old to new.

Document [1] was copied to [3] and document [2] was copied to [4]. The header sections were modified to match the new locations.
Document [1] was modified to a redirection file pointing to [3].
Document [2] was modified to a redirection file pointing to [4].

---
[1] Support/Help_Articles/Setting_up_a_Sailfish_Phone/README.md
[2] Support/Help_Articles/Setting_up_a_Sailfish_Phone_part2/README.md
[3] Support/Help_Articles/Managing_Sailfish_OS/Setting_up_a_Sailfish_Phone/
[4] Support/Help_Articles/Managing_Sailfish_OS/Setting_up_a_Sailfish_Phone_part2/

### CODE REVIEW

**Review by vigejolla — COMMENTED** [review]
nitpicking: Please limit the line length of the commit messages. As you can see in this PR they get truncated and are hard to read. In this case the "Set direction from old to new" should not be on the first line.

File: Support/Help_Articles/Setting_up_a_Sailfish_Phone/README.md [hunk 1]
```diff
@@ -1,36 +1,8 @@
 ---
-title: Setting up a Sailfish phone
-permalink: Support/Help_Articles/Setting_up_a_Sailfish_phone/
-parent: Help Articles
+title: Setting up a Sailfish Phone
+permalink: Support/Help_Articles/Setting_up_a_Sailfish_Phone/
```
Comment (vigejolla) [inline Support/Help_Articles/Setting_up_a_Sailfish_Phone/README.md:3]: Oh no, the old permalink had a small error: The "phone" did not have upper case P. In order for the redirect to work, we can't change it now. 
```suggestion
permalink: Support/Help_Articles/Setting_up_a_Sailfish_phone/
```

**Review by jpwalden — COMMENTED** [review]
Content looks good to me, Ville already spotted the upper/lower case on the link.

File: Support/Help_Articles/Setting_up_a_Sailfish_Phone_part2/README.md [hunk 2]
```diff
@@ -1,54 +1,8 @@
 ---
-title: Setting up a Sailfish phone part 2
-permalink: Support/Help_Articles/Setting_up_a_Sailfish_phone_part2/
-parent: Help Articles
+title: Setting up a Sailfish Phone part2
+permalink: Support/Help_Articles/Setting_up_a_Sailfish_Phone_part2/
```
[the remark this answers is not in the document]
Comment (jovirkku) [reply Support/Help_Articles/Setting_up_a_Sailfish_Phone_part2/README.md]: Lowercase 'phone' reverted to the permalinks in the README files at their old locations.

**Review by vigejolla — APPROVED** [review]

**Pull request merged**